README.md

```diff
@@ -1009,6 +1059,7 @@ handlers:
     formatter: json
     stream: ext://sys.stdout
     filters:
+      - type_filter
       - isotime
       - django
```

Comment:
Add a comment that the `type_filter` must be before django filter. I would also let `isotime` as first here in the example to better show that type_filter and django are related.